src/browser/modules/D3Visualization/components/Graph.tsx

```diff
@@ -99,6 +115,7 @@ export class GraphComponent extends React.Component<GraphProps, GraphState> {
       this.svgElement.current,
       measureSize,
       this.handleZoomEvent,
+      this.handleDisplayZoomWheelInfoMessage,
```

Comment:
Cannot change to onScroll since it is not called at all scrolls, only when a message should be displayed (will be even more clear after changed the logic in Visualization according to later comment that it should not be shown when a user is currently scoll zooming a graph). I am not sure if displayZoomWheelInfoMessage better, since displayZoomWheelInfoMessage sounds like a boolean and not a function callback. Handle gives a clue that it is a callback I think.